go/iot/protocol/Protocol.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protocol
+
+const (
+	OK                = 200
+	FAIL              = 404
+	UNREGISTERTAG     = 501
+	UNREGISTERMEASURE = 502
+)
+
+type MeasureArgument struct {
+	Name         string
+	Measurement string
```

Comment:
nit: consider renaming this to `MeasureValue`, since [Measurement](https://github.com/census-instrumentation/opencensus-go/blob/master/stats/measure.go#L95) = Measure + Value.